back/config/schemas/settings.schema.json.erb

```diff
@@ -1268,6 +1268,18 @@
         }
       },
 
+      "proposals_participation_method": {
```

Comment:
I added this in my PR. I wonder if the copy is clear enough 🤔

Reply:
Perhaps we could add a warning or so "DO NOT ENABLE UNLESS YOU KNOW WHAT YOU'RE DOING". I think we did something like that in the past once.

Reply:
Because at the moment, if we allow proposal phases on real platforms, even if it's just for testing, we could get issues in places like the moderations or email digests or xlsx sheets or so.